modules/goldilocks/varables.tf

```diff
@@ -0,0 +1,60 @@
+variable "namespaces" {
+  type        = set(string)
+  default     = ["default"]
+  description = "Goldilocks labels on your namespaces"
+}
+
+variable "create_metric_server" {
+  type        = bool
+  default     = true
+  description = "Create metric server"
+}
+
+variable "create_vpa_server" {
+  type        = bool
+  default     = true
+  description = "VPA configure in the cluster"
+}
+
+variable "zone_name" {
+  type        = string
+  description = "Domain Name"
+}
+
+variable "alb_certificate_arn" {
+  type        = string
+  default     = ""
+  description = "Domain Certificate ARN"
+}
+variable "alb_name" {
+  type    = string
+  default = "goldilocks-dashboard"
+}
+variable "hostname" {
+  type    = string
+  default = "goldilocks.example.com"
+}
+
+variable "userpoolarn" {
```

Comment:
why not to have userpool, userpoolclientid and userpooldomain in a single object type variable and not use local variable, something like
```terraform
variable "auth" {
    type = object({
      userPoolARN = string,
      userPoolClientID = string,
      userPoolDomain = string
   })
} 
```